app/src/main/res/xml/changelog_master.xml

```diff
@@ -2,6 +2,17 @@
 <changelog xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="UnusedResources">
 
+    <release
+        version="2.2.0"
+        versioncode="124">
+        <change>Switch map library to MapLibre - finally bringing Transportr back to F-Droid!</change>
```

Comment:
Technically, we didn't make it back, yet ;)

Reply:
Yeah it's the next unchecked bullet point in the first post, but we cannot continue without tagging the release :P

Reply:
And we will make it back (I hope)! :smile: